mitodl/open-discussions
# add channel members page

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #1932 

#### What's this PR do?

This adds a new page in the app which shows all members of a channel (powered by the search index) and adds a link to that view from the 'people' widget.

I also fixed a secondary issue with the click targets for the people widget editing.

#### How should this be manually tested?

go to a channel where you're a moderator and add a new people widget. in the list when you're searching for people you should be able to click anywhere to add the person to the list, not just on the 'add' button. compare with the behavior on master to see what it's fixing.

then, once you've created your widget, you should see a link that says 'See all members', which should take you do a new page, at `/c/${channelName}/members`, which shows you all the people in the channel. You might want to add some fake users in the django shell to fill out the page.

It loads 50 at a time using the infinite scroll component, so make sure that you have more than 50 members so you can test that out.

Note that this doesn't yet implement filtering, as that will require (I think anyway) some changes to the index, so I wanted to leave it off for now in the interest of keeping things simple.

#### Screenshots (if appropriate)

![seelalllasdfasdffasdfasdfasdf](https://user-images.githubusercontent.com/6207644/55175704-32a19b80-5156-11e9-9b75-2f9088a00ee9.png)
![seelalllasdfasdff](https://user-images.githubusercontent.com/6207644/55175705-333a3200-5156-11e9-887a-33e63a876eb6.png)
![seelalllasdf](https://user-images.githubusercontent.com/6207644/55175706-333a3200-5156-11e9-935a-9259e1e4172d.png)
